### ISSUE TITLE
Main nav and side nav active item should not be a link

### ISSUE THREAD

**Issue by alex-page** [govau/uikit#525]
# Bug Report

- [x] I’ve read and understood the [Contributing guidelines](https://github.com/govau/uikit/blob/master/CONTRIBUTING.md) and have done my best effort to follow them.
- [x] I’ve read and agree to the [Code of Conduct](https://github.com/govau/uikit/blob/master/CODE_OF_CONDUCT.md).
- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

Consistent experiences are not followed here based on work that was done with breadcrumbs.

## What I expected to happen

Main nav and side nav should have active items that are not links.

## Reproducing

- Module name: main-nav, side-nav
